iib/web/models.py

```diff
@@ -852,16 +857,10 @@ def from_json(cls, kwargs, batch=None):
                 '"bundles" should be either an empty array or an array of non-empty strings'
             )
 
-        # Check if no bundles `from_index and `binary_image` are specified
-        # if no bundles and and no from index then a empty index will be created
-        # if no binary image and just a from_index then we are not updating anything and it would
-        # be a no-op
-        if not request_kwargs.get('bundles') and (
-            not request_kwargs.get('from_index') or not request_kwargs.get('binary_image')
-        ):
-            raise ValidationError(
-                '"from_index" and "binary_image" must be specified if no bundles are specified'
-            )
+        # Check if no bundles and `from_index is specified
+        # if no bundles and and no from index then an empty index will be created which is a no-op
+        if not request_kwargs.get('bundles') and not request_kwargs.get('from_index'):
```

Comment:
Optional: It's more concise but I'll leave it up to you for readability
```suggestion
        if not (request_kwargs.get('bundles') and request_kwargs.get('from_index')):
```

Reply:
Ah, this should be `or` :p

That broke the build lol. I should have checked

Reply:
Ah correct